commcare_connect/program/views.py

```diff
@@ -277,3 +261,122 @@ def get_queryset(self):
         start_date = self.request.GET.get("start_date") or None
         end_date = self.request.GET.get("end_date") or None
         return get_delivery_performance_report(program, start_date, end_date)
+
+
+@org_member_required
+def program_home(request, org_slug):
+    org = Organization.objects.get(slug=org_slug)
+    is_program_manager = request.org.program_manager and (
```

Comment:
This should use the is_program_manager method that's already added

Reply:
Fixed. [67fe81f](https://github.com/dimagi/commcare-connect/pull/585/commits/67fe81f43013d02017e6d2467b2c4a7602f43154)